.changes/next-release/Breaking Change-df2d10e0-8c87-47ce-a038-56794a9abb6e.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Looks like some stale changes are trying to sneak in.

Reply:
Oh boy. Did an automerge PR get squash-merged in?

Reply:
Says it's from https://github.com/aws/aws-toolkit-vscode/pull/2219

Reply:
Thanks for noticing this. Opened a new PR at https://github.com/aws/aws-toolkit-vscode/pull/2326 since I can't force-push to `feature/ecs`.